packages/reference/src/components/LinkActions/LinkEntityActions.tsx

```diff
@@ -177,3 +178,16 @@ export function LinkEntityActions({
 
   return renderLinkActions(props);
 }
+
+export function CombinedLinkEntityActions({
+  renderCustomActions,
+  ...props
+}: LinkActionsProps & {
+  renderCustomActions?: (props: LinkActionsProps) => React.ReactElement;
+}) {
+  const renderLinkActions = renderCustomActions
+    ? renderCustomActions
+    : (props: LinkActionsProps) => <CombinedLinkActions {...props} />;
+
+  return renderLinkActions(props);
```

Comment:
Would this not work? I find it a bit more readable
```suggestion
  return renderCustomActions ? renderCustomActions(props) : <CombinedLinkActions {...props} />;
```

Reply:
Sure